test/gov/GovPool.test.js

```diff
@@ -972,6 +975,22 @@ describe("GovPool", () => {
         it("should revert when try move without vote", async () => {
           await truffleAssert.reverts(govPool.moveProposalToValidators(3), "Gov: can't be moved");
         });
+
+        it("should revert when validators count is zero", async () => {
```

Comment:
Please check in any test where there is a validators vote that the status becomes "executed" after the execution

Reply:
done